quantmsrescore/__init__.py

```
@@ -1,7 +1,234 @@
-# Apply warning filter before any imports
+# =============================================================================
+# Thread Control Configuration (MUST be done before importing heavy libraries)
+# =============================================================================
+# This module controls thread allocation for numerical libraries to prevent
+# thread explosion in HPC/Slurm environments where multiple processes compete
+# for resources.
+#
+# Problem: Libraries like PyTorch, TensorFlow, NumPy (via MKL/OpenBLAS) spawn
+# threads equal to CPU count by default. With multiprocessing, this causes:
+#   - processes × cpu_count threads competing for cpu_count cores
```

Ruff (0.14.10)

Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)